usr/src/common/zfs/zfs_prop.c

```diff
@@ -292,6 +293,10 @@ zfs_prop_init(void)
 	zprop_register_index(ZFS_PROP_NBMAND, "nbmand", 0, PROP_INHERIT,
 	    ZFS_TYPE_FILESYSTEM | ZFS_TYPE_SNAPSHOT, "on | off", "NBMAND",
 	    boolean_table);
+   zprop_register_index(ZFS_PROP_COMPRESS_QUOTA, "compress_quota", 0,
```

Comment:
There are style issues throughout these changes.  As a first pass, can you make sure that it passes the `cstyle -cpP` tool?  The automated tests should run that, so you can find it there once they complete.